src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -248,23 +248,23 @@ private Binder GetBinder(SyntaxNode syntax)
 
         private void NullableAnalyzeParameterDefaultValueFromAttributes()
```

Comment:
>NullableAnalyzeParameterDefaultValueFromAttributes [](start = 21, length = 50)

Could you point me to where do we analyze regular default values and attributes?

Reply:
Regular default values are handled at line 216 above: `NullableWalker.AnalyzeIfNeeded(binder, parameterEqualsValue, diagnostics);`

Attribute argument values are handled at Binder_Attributes.cs line 199: `NullableWalker.AnalyzeIfNeeded(this, boundAttribute, diagnostics);`

---
In reply to: [544697118](https://github.com/dotnet/roslyn/pull/49953#discussion_r544697118) [](ancestors = 544697118)

Reply:
I would prefer those to "explicitly" pass the context information. Simply pulling syntax from bound nodes is fragile and impossible to code review.

---
In reply to: [544701342](https://github.com/dotnet/roslyn/pull/49953#discussion_r544701342) [](ancestors = 544701342,544697118)